back-end/src/lms/views/absences.py

```diff
@@ -62,6 +65,27 @@ class AbsencePermission(BasePermission):
     ]
 
 
+class AbsenceAttachmentPermission(BasePermission):
+    permission_class = "absence-attachments"
+    view_name_rus = "Приложения"
```

Comment:
This name will be used in permission name. Add a bit of context:
```suggestion
    view_name_rus = "Приложения к пропускам"
```